spec/system/shared/admin_proposals_overdue_examples.rb

```diff
@@ -13,7 +13,8 @@
     let(:evaluating_days_overdue) { 0 }
 
     it "proposals are not highlighted" do
-      expect(page).not_to have_css(".help-text-overdue")
+      expect(page).not_to have_css(".help-text-overdue .text-warning")
+      expect(page).not_to have_css(".help-text-overdue .text-alert")
```

Comment:
```suggestion
      expect(page).not_to have_css(".help-text-overdue.text-warning")
      expect(page).not_to have_css(".help-text-overdue.text-alert")
```
What are you trying to accomplish here? if you want to check for the NON-existance of two classes in the same html element you should have no spaces in between. If you are trying to check an element with a class and also a children of that element with the second class, then the space is correct.

Option (no spaces):
```html
<div class="help-text-overdue text-success">...
```

Option 2 (spaces):
```html
<div class="help-text-overdue">
  <p class="text-success">...
```